drivers/spi/spi_ll_stm32.c

```diff
@@ -534,6 +829,8 @@ static const struct spi_stm32_config spi_stm32_cfg_##id = {		\
 static struct spi_stm32_data spi_stm32_dev_data_##id = {		\
 	SPI_CONTEXT_INIT_LOCK(spi_stm32_dev_data_##id, ctx),		\
 	SPI_CONTEXT_INIT_SYNC(spi_stm32_dev_data_##id, ctx),		\
+	SPI_DMA_CHANNEL_INIT(id, rx, RX, PERIPH, MEM)			\
+	SPI_DMA_CHANNEL_INIT(id, tx, TX, MEM, PERIPH)			\
```

Comment:
In order to compile properly I had to use PERIPHERAL and MEMORY.

Reply:
ok @FRASTM ,

I see that you fixed the defines instead.
OK, good, I'll test it!